src/Xamarin.Android.Build.Tasks/Utilities/NuGetLogger.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using NuGet.Common;
+using Microsoft.Build.Utilities;
+
+namespace Xamarin.Android.Tasks {
+
+	class NuGetLogger : ILogger {
+		TaskLoggingHelper log;
+
+		public NuGetLogger (TaskLoggingHelper log)
+		{
+			this.log = log;
+		}
+
+		public void Log (LogLevel level, string data)
+		{
+			log.LogDebugMessage (data);
```

Comment:
This is an exception in waiting: if `data` contains e.g. `{foo}`, then it'll throw, because [`MSBuildExtensions.LogDebugMessage()`](https://github.com/xamarin/xamarin-android/blob/e73b18e70ac4a9b66e1785c8bf185a1ebc667265/src/Xamarin.Android.Build.Tasks/Utilities/MSBuildExtensions.cs#L23-L26) will interpret `data` as a *format string*, and `{foo}` is *not* a valid format string.

There are two plausible fixes:

1. We could (should!) overload `MSBuildExtensions.LogDebugMessage()` with a `LogDebugMessage(this TaskLoggingHelper log, string message)` overload so I can stop looking for these issues (something we should do for *every* log/write/etc. method!), or
2. We should provide an actual format string.

For example:

    log.LogDebugMessage("{0}", data);

will ensure that things work sanely no matter what `data` contains.